src/lib.rs

```diff
@@ -160,10 +166,34 @@ pub fn run(working_dir: &Path, args: Cli) -> anyhow::Result<()> {
     Ok(())
 }
 
-fn project_manifest(manifest_path: &Path, cargo: context::Cargo) -> anyhow::Result<PathBuf> {
-    let manifest = metadata::workspace_root(manifest_path, cargo)?.join(CARGO_TOML);
+fn remove_override(
+    Context {
+        manifest_path,
+        manifest_dir,
+        working_dir,
+        cargo: _,
+        force: _,
+        operation,
+        registry_hint: _,
+    }: Context,
+) -> anyhow::Result<()> {
+    let package = match operation {
+        context::Operation::Remove { name } => name,
+        _ => unreachable!(),
+    };
 
-    debug_assert!(manifest.is_file(), "{:?} is not a file", manifest);
+    let project_manifest_content =
+        fs::read_to_string(&manifest_path).context("failed to read patch manifest")?;
+
+    let project_manifest_toml = toml::patch_manifest(
+        working_dir,
+        project_manifest_content.as_str(),
+        manifest_dir.as_path(),
+        toml::Operation::Remove { name: &package },
+    )?;
 
-    Ok(manifest)
+    fs::write(&manifest_path, &project_manifest_toml)
+        .context("failed to write patched `Cargo.toml` file")?;
+
```

Comment:
We should probably print something out when the removal was successful.

Something similar to what we do for adding patches.

https://github.com/eopb/cargo-override/blob/7050e7733e437fcf425b9a2d8f48376312efe76c/src/lib.rs#L161-L164